proto/api/v1/config_service.proto

```diff
@@ -53,6 +53,10 @@ message UpdateVulnerabilityExceptionConfigResponse {
   VulnerabilityExceptionConfig config = 1;
 }
 
+message PutPlatformComponentConfigRequest {
```

Comment:
I understand why you named this `Put`, but I think `Update` matches the other endpoints.

Reply:
The "Put" vs "Update" terminology is even more confused than that.

Based on an internal support question about our API docs, I have discovered that, in the imageintegration service, we [use "Put" for PUT](https://github.com/stackrox/stackrox/blob/master/central/imageintegration/service/service_impl.go#L148-L151), and ["Update" for PATCH](https://github.com/stackrox/stackrox/blob/master/central/imageintegration/service/service_impl.go#L148-L151). 

What's worse, we don't use PATCH there to replace partial properties, but for the stated reason ", with optional stored credential reconciliation". 😱